packages/google-analytics/src/index.ts

```diff
@@ -59,19 +41,17 @@ const googleAnalytics: GoogleAnalytics = {
             "Trying to send an event to Google Analytics but neither `trackingId` nor `trackingIds` are specified inside `state.googleAnalytics`."
           );
 
-        ids
-          .map((id) => getTrackerName(id))
-          .forEach((trackerName) => {
-            const { category, label, value, ...rest } = payload;
-            window.ga(`${trackerName}.send`, {
-              hitType: "event",
-              eventAction: name,
-              eventCategory: category,
-              eventLabel: label,
-              eventValue: value,
-              ...rest,
-            });
+        ids.forEach((id: any) => {
+          const { category, label, value, ...rest } = payload;
+
+          window.gtag("event", name, {
+            send_to: id,
+            event_category: category,
+            event_label: label,
```

Comment:
It doesn't seem like GA4 is using action, category and label anymore, is it? https://developers.google.com/analytics/devguides/collection/ga4/events#recommended_events

As this is a major bump, maybe we should change the event to something that makes more sense for the new events.

Reply:
Oh, hmm 🤔  this is odd. I was using this guide https://developers.google.com/analytics/devguides/collection/upgrade/analyticsjs#measure_events which has a before and after examples.

Reply:
This new implementation, with the new library, supports also Universal Analytics (previous to GA4). I think that's the reason they keep sending `event_category` and `event_label`. Most of the users are currently using Universal Analytics, so I guess we should keep it working as well.

Reply:
But what if a user wants to send one of the new events? Shouldn't we at least support both?

Anyway, let's talk it over https://community.frontity.org/t/google-analytics-package/1084/16.

Reply:
I was writing in https://community.frontity.org/t/google-analytics-package/1084/16 to talk about the implementation of this, but I've just realized that the implementation that Cristian did works for both cases (old and new) thanks to the `...rest` spread in the event:

```js
const { category, label, value, ...rest } = payload;

window.gtag("event", name, {
  send_to: id,
  event_category: category,
  event_label: label,
  value,
  ...rest, // <-- adds any other payload
});
```

If an event contains `category` or `label`, it would be translated to `event_category` and `event_label`, and if not, it will be passed down as part of `...rest`.

So sending the new events (things like this: https://support.google.com/analytics/answer/9268036) should work, shouldn't it?